app/types.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'vanilla-text-mask'
```

Comment:
I installed the `vanilla-text-mask` and it doesn't use typescript, so I needed to declare this module to fake its types declaration